src/components/Drawer/Drawer.tsx

```diff
@@ -39,26 +39,33 @@ const DrawerWrapper = styled.div`
 `;
 
 const Header = styled(Padbox)`
-  border-bottom: 1px solid ${getColor('neutral.500')};
+  border-bottom: 1px solid ${getColor('neutral.400')};
 `;
 
 const BaseDrawer = styled.div<{ $maxWidth: number }>`
   display: flex;
   flex-direction: column;
-  height: 100%;
+  height: calc(100% - ${getSpace('lgPlus')});
   width: 100%;
+  border-radius: ${getRadii('default')};
+  margin: ${getSpace('mdPlus')};
   max-width: ${({ $maxWidth }) => pxToRem($maxWidth)};
   background-color: ${getColor('neutral.0')};
-  box-shadow: -4px 0px 16px rgba(0, 0, 0, 0.1);
+  box-shadow: 0px 0px 32px 16px rgba(0, 0, 0, 0.1);
 `;
+// box-shadow: -4px 0px 16px rgba(0, 0, 0, 0.1);
 
 const TitleWrapper = styled(Padbox)`
   overflow: hidden;
 `;
 
 /* stylelint-disable */
-const Title = styled(H4)`
+const Title = styled(Text).attrs({
+  as: 'h1',
```

Comment:
This shouldn't be `h1`. I would personally use `p` tag as this is mostly side content which is not part of the outline document